src/textual/geometry.py

```diff
@@ -905,6 +905,47 @@ def translate_inside(
             height2,
         )
 
+    def inflect(
+        self, x_axis: int = +1, y_axis: int = +1, margin: Spacing | None = None
+    ) -> Region:
+        """Inflect a region around one or both axis.
+
+        The `x_axis` and `y_axis` parameters define which direction to move the region.
+        A positive value will move the region right or down, a negative value will move
+        the region left or up.
```

Comment:
Should we specify that the value `0` does _not_ inflect the given axis?

Reply:
I guess so.